src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -113,220 +117,227 @@ public void BuildDeltaEmptyPoolContent()
+        //            i > 10 ? (uint)DeltaGasLimit / 8U - 10000U : 70000U, // to test scenarios when both single transaction is ignored and all remaining
+        //            (20 + i).GFul(),
+        //            Bytes.Empty,
+        //            receiverPublicKey: i.ToString(),
+        //            transactionFees: (ulong)_random.Next(),
+        //            timestamp: _random.Next(),
+        //            signature: i.ToString());
+        //        return MempoolHelper.GetMempoolItems(transaction, _hashProvider);
+        //    }).SelectMany(x => x).ToList();
+
+        //    var transactionRetriever = BuildRetriever(transactions);
+        //    var expectedSelectedTransactions = BuildSelectedTransactions(transactions.Skip(10).Take(1).Union(transactions.Skip(12).Take(8)).ToList());
+        //    var publicEntries = expectedSelectedTransactions.Select(x => x.ToProtoBuff<MempoolItem, PublicEntry>(_mapperProvider));
+
+        //    var salt = BitConverter.GetBytes(
+        //        _randomFactory.GetDeterministicRandomFromSeed(_previousDeltaHash.ToArray()).NextInt());
+
+        //    var rawAndSaltedEntriesBySignature = publicEntries.Select(e =>
+        //        {
+        //            var contractEntriesProtoBuff = e;
+        //            return new
+        //            {
+        //                RawEntry = contractEntriesProtoBuff,
+        //                SaltedAndHashedEntry =
+        //                    _hashProvider.ComputeMultiHash(contractEntriesProtoBuff.ToByteArray().Concat(salt))
+        //            };
+        //        }).ToArray();
+
+        //    var shuffledEntriesBytes = rawAndSaltedEntriesBySignature
+        //       .OrderBy(v => v.SaltedAndHashedEntry.ToArray(), ByteUtil.ByteListComparer.Default)
+        //       .SelectMany(v => v.RawEntry.ToByteArray())
+        //       .ToArray();
+
+        //    var expectedBytesToHash = BuildExpectedBytesToHash(expectedSelectedTransactions, shuffledEntriesBytes);
+
+        //    var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
+        //        _cache, _dateTimeProvider, _mapperProvider, _logger);
+        //    var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
+
+        //    ValidateDeltaCandidate(candidate, expectedBytesToHash);
+
+        //    _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
+        //}
+
+        private static PublicEntry[] BuildSelectedTransactions(List<PublicEntry> transactions)
         {
-            var selectedTransactions = transactions.Where(t => (t.IsPublicTransaction || t.IsContractCall || t.IsContractDeployment) && t.HasValidEntries()).ToArray();
-            return selectedTransactions;
+            return transactions.ToArray();
+            //var selectedTransactions = transactions.Where(t => (t.IsPublicTransaction || t.IsContractCall || t.IsContractDeployment) && t.HasValidEntries()).ToArray();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)